_non-cosmos/ethereum/assetlist.json

```diff
@@ -2520,6 +2520,29 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/susds.svg"
       },
       "coingecko_id": "susds"
+    },
+    {
+      "type_asset": "erc20",
+      "address": "0x007115416ab6c266329a03b09a8aa39ac2ef7d9d",
+      "denom_units": [
+        {
+          "denom": "0x007115416ab6c266329a03b09a8aa39ac2ef7d9d",
+          "exponent": 0
+        },
+        {
+          "denom": "mBTC",
+          "exponent": 18
+        }
+      ],
+      "base": "0x007115416ab6c266329a03b09a8aa39ac2ef7d9d",
+      "name": "Midas BTC Yield Token",
+      "display": "mBTC",
+      "symbol": "mBTC.midas",
```

Comment:
why does the symbol have '.midas'? I assume that's what the prefix 'm-' stands for.